app/services/data/tear-down/public-schema.service.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+/**
+ * Removes all data created for acceptance tests from the public schema
+ * @module PublicSchemaService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+async function go () {
+  return _deleteAllTestData()
+}
+
+async function _deleteAllTestData () {
+  return db.raw(`
+  ALTER TABLE public.review_returns DISABLE TRIGGER ALL;
+  ALTER TABLE public.review_charge_elements_returns DISABLE TRIGGER ALL;
+  ALTER TABLE public.review_charge_elements DISABLE TRIGGER ALL;
+  ALTER TABLE public.review_charge_references DISABLE TRIGGER ALL;
+  ALTER TABLE public.review_charge_versions DISABLE TRIGGER ALL;
+  ALTER TABLE public.review_licences DISABLE TRIGGER ALL;
```

Comment:
My two-cents!

We need to disable these triggers on the legacy tables because they set up foreign key constraints. This means we can't delete record `x` because it's linked to `y` in a different table.

Unless I missed something, we didn't complicate things by adding any foreign key constraints so you _shouldn't_ need to disable and re-enable any triggers.

Reply:
That's cool, Ive dropped those now. Are you happy with me using the test region to identify the test data rather than adding an 'is_test' column to the reviewLicences table?